src/main.ts

```diff
@@ -258,21 +133,11 @@ async function home(motionClient: MotionClient, armClient: ArmClient) {
     pose: home_pose
   }
 
-  await motionClient.move(home_pose_in_frame, myResourceName, myWorldState, constraints)
+  await motionClient.move(home_pose_in_frame, armName, myWorldState, constraints)
 }
 
 async function moveToQuadrant(motionClient: MotionClient, armClient: ArmClient, x: number, y: number) {
-    //Create a Worldstate that has the GeometriesInFrame included 
-    let myObstaclesInFrame: SDK.GeometriesInFrame = {
-      referenceFrame: "world", 
-      geometriesList: [frontWallObject, backWallObject, leftWallObject, rightWallObject],
-    }
-    
-    let myWorldState: SDK.WorldState ={
-      obstaclesList: [myObstaclesInFrame],
-      transformsList: [],
-    }
-  
+  if (ignoreInterrupts && await armClient.isMoving()) { return }
```

Comment:
[q] is the `armClient.isMoving()` check required? I thought that buttons would be "disabled" if we are still planning/executing, i.e. we have not returned a response success or response failure from the outgoing ms.move call.

Reply:
In theory it shouldn't be required - but if someone were moving the arm via a script or other means, this would safeguard against interrupting that move?
I think its fine to leave in and good to add here to be consistent with other move methods